examples/tutorial-standalone/main.tf

```diff
@@ -0,0 +1,198 @@
+    google_project_iam_binding.remove_owner_role
+  ]
+}
+
+module "de_identification_template" {
+  source = "../..//modules/de-identification-template"
+
+  project_id                = module.base_projects.data_governance_project_id
+  terraform_service_account = var.terraform_service_account
+  crypto_key                = module.tek_wrapping_key.keys[local.kek_key_name]
+  wrapped_key               = local.wrapped_key_secret_data
+  dlp_location              = local.location
+  template_id_prefix        = "de_identification"
+  template_file             = "${path.module}/templates/deidentification.tmpl"
+  dataflow_service_account  = module.secured_data_warehouse.dataflow_controller_service_account_email
+}
+
+module "re_identification_template" {
+  source = "../..//modules/de-identification-template"
+
+  project_id                = module.base_projects.data_governance_project_id
+  terraform_service_account = var.terraform_service_account
+  crypto_key                = module.tek_wrapping_key.keys[local.kek_key_name]
+  wrapped_key               = local.wrapped_key_secret_data
+  dlp_location              = local.location
+  template_id_prefix        = "re_identification"
+  template_file             = "${path.module}/templates/reidentification.tmpl"
+  dataflow_service_account  = module.secured_data_warehouse.confidential_dataflow_controller_service_account_email
+}
+
+resource "google_artifact_registry_repository_iam_member" "docker_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "flex-templates"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_docker_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "flex-templates"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.dataflow_controller_service_account_email}"
+}
+
+resource "google_artifact_registry_repository_iam_member" "confidential_python_reader" {
+  provider = google-beta
+
+  project    = module.template_project.project_id
+  location   = local.location
+  repository = "python-modules"
+  role       = "roles/artifactregistry.reader"
+  member     = "serviceAccount:${module.secured_data_warehouse.confidential_dataflow_controller_service_account_email}"
+}
+
+module "regional_deid" {
```

Comment:
could we add `pipeline` to the name and also to the re-id one. I think it'll be helpful to have that mental construct that this  creates that batch job to person the actual ingestion.